plutus-tx/testlib/PlutusTx/Test.hs

```diff
@@ -227,3 +228,12 @@ runPlcCekTrace values = do
         UPLC.runCek PLC.defaultCekParameters UPLC.tallying UPLC.logEmitter (p ^. UPLC.progTerm)
   res <- fromRightM (throwError . SomeException) result
   pure (logOut, tally, res)
+
+-- | A group of golden tests including UPLC size and UPLC and PIR readable.
+goldenTests :: String -> CompiledCode a -> [TestTree]
+goldenTests testName code =
+  [
+  runTestNested $ goldenSize testName code,
```

Comment:
You should actually run them in a directory, look where the golden files have been generated. You want something like `runTestNestedIn ["test", "size"]`

Reply:
Ah, or rather you want this to return `TestNested` so the size tests can discharge that correctly